pkg/limayaml/defaults.go

```diff
@@ -450,6 +451,22 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		env[k] = v
 	}
 	y.Env = env
+
+	if y.CACertificates.RemoveDefaults == nil {
+		y.CACertificates.RemoveDefaults = d.CACertificates.RemoveDefaults
+	}
+	if o.CACertificates.RemoveDefaults != nil {
+		y.CACertificates.RemoveDefaults = o.CACertificates.RemoveDefaults
+	}
+	if y.CACertificates.RemoveDefaults == nil {
+		y.CACertificates.RemoveDefaults = pointer.Bool(false)
+	}
+
+	caFiles := unique(append(append(d.CACertificates.Files, y.CACertificates.Files...), o.CACertificates.Files...))
+	y.CACertificates.Files = caFiles
+
+	caCerts := unique(append(append(d.CACertificates.Certs, y.CACertificates.Certs...), o.CACertificates.Certs...))
+	y.CACertificates.Certs = caCerts
```

Comment:
It would probably be better to check for duplicates ones all the files have been read and combined with the literal certs. You might have multiple files containing the same cert, or having a file and a literal entry be the same. Or you might have `~/ca.pem` and `/Users/jan/ca.pem`, which would be the same file, but not recognized as duplicates.

On the other hand I don't think having duplicate certs is a problem, so not marking this as a blocker.